package.json

```diff
@@ -8,7 +8,7 @@
     "format:write": "yarn prettier --write .",
     "lint": "echo 'Linting all packages' && wsrun --parallel --exclude-missing lint",
     "test": "echo 'Running tests' && wsrun --parallel --exclude-missing test",
-    "build-common": "echo 'Building all common packages packages' && wsrun --stages --exclude-missing build-pre-app",
+    "build-common": "echo 'Building all common packages packages' && wsrun --stages --exclude @ehpr/api --exclude @ehpr/web --exclude-missing build",
```

Comment:
Last comment, we can fix this later but I'd like the watch script to follow this pattern as well, being able to explicitly say "watch all packages except API/Web"